$NetBSD$

Honor LDFLAGS and the configured strip command.  Name object outputs
explicitly for VSI C.

--- deformatters/Makefile.orig
+++ deformatters/Makefile
@@ -80,17 +80,17 @@
 defmt-c: defmt-c.o
 	@. ../config.sh; \
 	  set -x; \
-	  $$CC $$CFLAGS -o $@ $<
+	  $$CC $$CFLAGS $$LDFLAGS -o $@ $<
 
 defmt-sh: defmt-sh.o
 	@. ../config.sh; \
 	  set -x; \
-	  $$CC $$CFLAGS -o $@ $<
+	  $$CC $$CFLAGS $$LDFLAGS -o $@ $<
 
 .c.o:
 	@. ../config.sh; \
 	  set -x; \
-	  $$CC $$CFLAGS -c $<
+	  $$CC $$CFLAGS -c -o $@ $<
 
 install: all
 	@. ../config.sh; \
@@ -103,7 +103,7 @@
 	@. ../config.sh; \
 	  set -x; \
 	  cd $(DESTDIR)$$BINDIR; \
-	  strip $(PROGRAMS); \
+	  $(STRIP) $(PROGRAMS); \
 	  chmod 755 $(PROGRAMS)
 
 clean:
